recipes/libcurl/all/conanfile.py

```diff
@@ -5,7 +5,7 @@
 from conans import ConanFile, AutoToolsBuildEnvironment, CMake, tools
 from conans.errors import ConanInvalidConfiguration
 from conan.tools.microsoft import is_msvc
-from conan.tools.build import cross_building
+from conan.tools.build.cross_building import cross_building
 
 required_conan_version = ">=1.45.0"
```

Comment:
Should the version be bumped? I image there a release where that was changed

Reply:
Yes, please.

Reply:
I did this change, but now other things break..